core/src/main/scala/kafka/server/BrokerToControllerChannelManager.scala

```diff
@@ -113,8 +113,8 @@ class BrokerToControllerChannelManager(metadataCache: kafka.server.MetadataCache
       brokerToControllerListenerName, time, threadName)
   }
 
-  private[server] def sendRequest(request: AbstractRequest.Builder[_ <: AbstractRequest],
-                                  callback: RequestCompletionHandler): Unit = {
+  def sendRequest(request: AbstractRequest.Builder[_ <: AbstractRequest],
+                  callback: RequestCompletionHandler): Unit = {
     requestQueue.put(BrokerToControllerQueueItem(request, callback))
```

Comment:
This also needs to call `NetworkClient#wake` in case we are blocking inside `NetworkClient#poll`